server/service/integration_enterprise_test.go

```diff
@@ -4034,9 +4034,12 @@ func (s *integrationEnterpriseTestSuite) TestGitOpsUserActions() {
 	// Attempt to retrieve activities, should fail.
 	s.DoJSON("GET", "/api/latest/fleet/activities", nil, http.StatusForbidden, &listActivitiesResponse{})
 
-	// Attempt to retrieve hosts, should fail.
+	// Attempt to retrieve hosts, should succeed.
```

Comment:
I think the original comment is still ok, it should `fail`? It still expects a failure status code.

Reply:
thank you!! this is a residue of another approach I was trying